tests/test_dryer.py

```diff
@@ -0,0 +1,88 @@
+from collections.abc import Callable
+from typing import Any
+
+import pytest
+from aioresponses import aioresponses
+from yarl import URL
+
+from whirlpool.appliancesmanager import AppliancesManager
+from whirlpool.auth import Auth
+from whirlpool.backendselector import BackendSelector
+from whirlpool.dryer import MachineState
+
+
+async def test_attributes(appliances_manager: AppliancesManager):
+    dryer1 = appliances_manager.dryers[0]
```

Comment:
```suggestion
    dryer = appliances_manager.dryers[0]
```
there is only 1 instance